apps/site/src/app/schedule/DayOne/DayOne.tsx

```diff
@@ -24,6 +24,7 @@ const DayOne: React.FC<DayOneProps> = ({ schedule, time }) => {
 	return (
 		<>
 			<div className={styles.mapContainer}>
+				<h3 className="visually-hidden">Day 1</h3>
```

Comment:
Thought: doesn't the map already include text for the day? Couldn't we turn that into a heading? (same for event title)